null/pipeline_info/execution_trace_2023-01-06_10-31-12.txt

```diff
@@ -0,0 +1 @@
+task_id	hash	native_id	name	status	exit	submit	duration	realtime	%cpu	peak_rss	peak_vmem	rchar	wchar
```

Comment:
I'd probably add the `pipeline_info` stuff to the `.gitignore` as well and remove this.